src/data-workspace/inputs/generic-input.js

```diff
@@ -43,23 +43,51 @@ export const GenericInput = ({
         )
     }
 
+    const formatValue = (value) => {
+        let formattedValue = value
+
+        if (value && valueType === VALUE_TYPES.NUMBER) {
+            if (value.startsWith('.')) {
```

Comment:
I think this will lead to `.` interpreting to `0.` on blur?

Reply:
Yes doing this onBlur is where it would happen, but maybe only doing it if there were additional digits following would have been good to. I think a more general use of `Number().toString()` like you suggest in the comment below would cover this too.